test/kata4.humanCatDogYears.test.js

```diff
@@ -1,3 +1,11 @@
+
 const { humanCatDogYears } = require("../src");
 
-// Look Ma, no handlebars!!!
+describe("humanCatDogYears", () => {
+    test("returning array of human, cat and dog years", () => {
```

Comment:
Everywhere else you are using "it" instead of "test", for better consistency use this or that only :)
And too much indentation, Line 5, 7, 8, and then needs one tab or two spaces,
and line 6, and 9 the expects need 4 spaces or 2 tabs.